invenio_rdm_records/services/tasks.py

```diff
@@ -27,3 +45,53 @@ def update_expired_embargos():
         except EmbargoNotLiftedError as ex:
             current_app.logger.warning(ex.description)
             continue
+
+
+@shared_task(ignore_result=True)
+def reindex_stats(stats_indices, max_bucket_size=10000):
+    """Reindex the documents where the stats have changed."""
+    bm = BookmarkAPI(current_search_client, "stats_reindex", "day")
+    last_run = bm.get_bookmark()
+    bookmark_timestamp = bm.prepare_bookmark()
+    indices = ",".join(map(lambda x: prefix_index(x) + "*", stats_indices))
+    query = dsl.Search(
+        using=current_search_client,
+        index=indices,
+    ).extra(size=0)
+    if last_run:
+        query = query.filter({"range": {"updated_timestamp": {"gte": last_run}}})
```

Comment:
Note that in the first execution it will reindex everything!

Reply:
We need to discuss this :)

Reply:
Let's put one week ago